openedx/features/offline_mode/utils.py

```diff
@@ -36,6 +37,8 @@ def generate_offline_content(xblock, html_data):
     try:
         save_xblock_html(tmp_dir, xblock, html_data)
```

Comment:
We should rename this function and call it sanitize or prepare xblock html data

Reply:
But this function actually saves the rendered xblock html to the index.html file.

Reply:
Ok, let's leave it for now